.github/workflows/add-to-board.yml

```
@@ -21,12 +25,16 @@ jobs:
     env:
       PROJECT_ID_MAP: |
         {
-          "team:DocsEng":   "1625",
-          "team:Obs":       "649",
-          "team:Platform":  "1232",
-          "team:Projects":  "1415",
-          "team:Search":    "726",
-          "team:Security":  "1034"
+          "team:Ingest":      "649",
+          "team:Experience":  "1034",
+          "team:Admin":       "1232",
```

https://github.com/orgs/elastic/projects/1927 please :)

```suggestion
          "team:Admin":       "1927",
```